internal/budgetadjustments/events/examples/delete_example.sh

```diff
@@ -1,11 +1,26 @@
-# Delete Adjustment Evens using a file
-sloctl budgetadjustments events delete --adjustment-name=sample-adjustment-name -f /path/to/file/with/events
+# Delete Adjustment Events using a file
+sloctl budgetadjustments events delete --adjustment-name=sample-adjustment-name -f /path/to/file/with/events/yaml
 
-# Delete Adjustment Event using stdin:
+# Delete Adjustment Events using stdin:
 echo '
```

Comment:
to make this whole example file more concise if you move the below example up:
same goes for update

```sh
# Delete Adjustment Events using a file:
cat <<EOF > ./events.yaml
- eventStart: 2024-10-24T04:07:04Z
  eventEnd: 2024-10-24T05:27:04Z
  slos:
  - project: test-project
    name: sample-slo-1
- eventStart: 2024-10-25T04:07:04Z
  eventEnd: 2024-10-25T05:27:04Z
  slos:
  - project: test-project
    name: sample-slo-2
EOF
sloctl budgetadjustments events delete --adjustment-name=sample-adjustment-name -f ./events.yaml

# Delete Adjustment Events using stdin:
sloctl budgetadjustments events delete --adjustment-name=sample-adjustment-name -f - <./events.yaml
```

Reply:
rearranged